### PR TITLE
Replace architecture principles with placeholder

### PR DESCRIPTION
**Pull request by himal-mandalia**
Removes architecture principles for now as the existing ones were too broad in their applicability. New principles in progress currently.

### CODE REVIEW

File: principles/development-principles.md [hunk 1]
```diff
@@ -5,34 +5,4 @@ expires: 2019-07-01
...
-
-### 5. Build for interoperability
-
-Services should be built using open standards where possible, and designed to be interoperable with other components.  We prefer loose coupling of components which allows the replacement of components at a later date as other options become available.
-
-### 6. Make boring technology decisions
-
-Boring technologies are technologies which are robust, well tested, well understood and just work.  We prefer the use of boring technologies within digital services unless there is an overriding reason for a specific technology.
-We encourage teams building production services to consider having "3 innovation tokens", which allows them to pick up to 3 areas in which to experiment or innovate.  These tokens must also take into account innovative service design, innovative support models or unknown business areas as well as technology innovations.
+Placeholder. Older principles reomved while new ones are being worked on. The ones previously here were found to be difficult to work with and too broad.
```
Comment (tvararu) [inline principles/development-principles.md]:
```suggestion
Placeholder. Older principles removed while new ones are being worked on. The ones previously here were found to be difficult to work with and too broad.
```

Comment (adrianclay) [inline principles/development-principles.md]: Typo with `reomved`.  I'd also like to know more about:
> ones previously here were found to be difficult to work with and too broad

Comment (luke-slowen) [reply principles/development-principles.md]: Yep, happy with that.

Comment (luke-slowen) [reply principles/development-principles.md]: Trying to get the right balance between the enterprise-level architecture principles and building out those more applicable at a technical level. Happy to get you involved in the conversation.

Comment (himal-mandalia) [reply principles/development-principles.md]: @adrianclay will update you in person in our catch up.

Comment (himal-mandalia) [reply principles/development-principles.md]: @luke-slowen would be good to involve @adrianclay and other tech leads in wider conversations around principles. Socialising via Slack is a good way of getting the conversation going, happy to help with that.

**Pull request merged**